src/openfermion/utils/_slater_determinants_test.py

```diff
@@ -448,26 +441,11 @@ def test_bad_input(self):
             givens_matrix = jw_sparse_givens_rotation(4, 5, 1., 1., 5)
 
 
-def random_quadratic_hamiltonian(n_qubits,
-                                 conserves_particle_number=False,
-                                 real=False):
-    """Generate a random instance of QuadraticHamiltonian
-
-    Args:
-        n_qubits(int): the number of qubits
-        conserves_particle_number(bool): whether the returned Hamiltonian
-            should conserve particle number
-        real(bool): whether to use only real numbers
-
-    Returns:
-        QuadraticHamiltonian
-    """
-    constant = numpy.random.randn()
-    chemical_potential = numpy.random.randn()
-    hermitian_mat = random_hermitian_matrix(n_qubits, real)
-    if conserves_particle_number:
-        antisymmetric_mat = None
+def random_unitary_matrix(n, real=False):
```

Comment:
I wonder if this should go in a more general module. Perhaps somewhere else in utils? Because this clearly has nothing to do with Slater determinants, per se, and could be useful in other contexts. Otherwise the PR is good to go.

Reply:
I had similar thoughts when I wrote the other functions that generate random objects for testing, too. I don't think they fit in any file that already exists. What if we put them in a new module called something like `utils/testing_utils.py`?

Reply:
That sounds fine to me. There is also a function in "performance_benchmarks.py" (in examples) called "artificial_molecular_operator" which could probably go in such a module. I think the docs still reference MolecularOperator (so perhaps that should also be updated). Feel free to rename that as you see fit.

Reply:
Actually, you might consider putting this in tests/testing_utils.py. I don't feel strongly though.